bindata/network/kuryr/003-config.yaml

```diff
@@ -73,6 +73,11 @@ data:
     user_domain_name = {{ default "\"\"" $AuthInfo.UserDomainName }}
     user_domain_id = {{ default "\"\"" $AuthInfo.UserDomainID }}
     region_name = {{ default "\"\"" .OpenStackCloud.RegionName }}
+{{ if .UserCACertificate }}
+    # There's no good way to just "append" user-provided certs to system ones,
+    # so just configure openstacksdk to use it.
+    cafile = /etc/ssl/certs/user-ca-bundle.crt
+{{ end }}
```

Comment:
pro tip: if you put a `-` after the `{{` here and on the `{{ end }}` line, it will delete the extra blank lines you'd otherwise get

Reply:
Thanks, I fixed all the places.